zio-http/src/main/scala/zhttp/service/Server.scala

```diff
@@ -133,24 +140,26 @@ object Server {
     keepAlive: Boolean = true,
     consolidateFlush: Boolean = false,
     flowControl: Boolean = true,
+    httpRequestDecompression: (Boolean, Boolean) = null,
```

Comment:
I think it's better we define it as null than allocate a new Tuple`(Bolean, Boolean)` that would be used in the `ServerChannelInitializer`